redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,17 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason is None:
+            reason = _("No reason was given.")
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle:
+            if guild.me.top_role > user.top_role and user != guild.owner and author != user:
```

Comment:
This is already checked above, no need to repeat this if.

Reply:
Which bit here is repeated?

Reply:
This is already done in line 172:
```py
if guild.me.top_role > user.top_role and user != guild.owner and author != user:
```

Reply:
You only removed one of those ifs, the one mentioned here is still not removed :)